### DISCUSSION TITLE
Alternative for Pi 4 8G

### DISCUSSION THREAD

**Discussion by Ilangovansub** [geezacoleman/OpenWeedLocator#39] [Ideas]
As most of us are ware on the low availablity of SCBs in the market. Is is possible to run this code on some unused laptops??
Old laptop with SDD and 8Gb doing all the processing and a board like Esp32 capturing the image and sending it to the laptop for processing and taking action as required?? Just thinking aloud :)

**Comment by geezacoleman** [comment]
That's a great question! I've seen it discussed on a few different RPi forums. I haven't looked too much into it myself - the biggest issue would be replacing the GPIO pins and also keeping things small.

[With these USB relay control boards](https://www.robotgear.com.au/Product.aspx/Discontinued/666-2-Channel-Relay-Module-with-USB-Interface-USB-RLY02) you could fairly quickly use a laptop to control the solenoids with a USB camera. It would just need a change to the relay control code, which wouldn't be too bad.

Laptops might not be rugged enough either, but perhaps you could leave it in the tractor and run cables to the actual device. Could be worth giving it a try if you have one lying around and wanted to test it though.

**Reply by Ilangovansub** [reply]
Thanks for your input. I am sure we can port most of the code to work on unused devices like laptops and tablets/phones. Since this will be mounted on a farm machinery size may not be a big factor. We may even be able to provide better (bigger) user interface to the farmer ;)
